Porting/vote_admin_guide.pod

```diff
@@ -0,0 +1,175 @@
+=encoding utf8
+
+=head1 NAME
+
+vote_admin_guide - Perl Governance Vote Administrator Guide
+
+=head1 Running a Steering Council nomination process
+
+Announce the nomination period to the Perl Core mailing list.  Be explicit
+about the end time.  Set a calendar reminder for yourself for when it's
+time to close the nominations and begin the voting.
+
+If someone outside the Core Team is nominated, contact them to confirm that
+they accept the nomination.  If they do, ask the Steering Council to add
+them as a moderated member of the Perl Core list for the duration of the
+election and invite them to offer a statement on their candidacy to the
+list.
+
+Remind inactive Core Team members that they have until the end of
+nominations to reactivate themselves if they wish to vote.
+
+Before the end of the nomination period, remind everyone of the schedule
+and share the current list of nominees to confirm you aren't missing
+anyone.
+
+At the end of the nomination period, notify the Perl Core list of the list
+of nominees and set their expectations for when you'll be opening the
+voting period.
+
+=head1 Using CIVS
+
+We are using L<Condorcet Internet Voting Service|https://civs.cs.cornell.edu/>
+(CIVS), which is pretty easy to use, although it has a lot of options.
+
+First, bookmark that link, but realize that there are no user accounts.
+Whenever you create or participate in a poll, you'll get a unique URL, and you
+need to save it to come back to what you were doing before.
+
+=head2 Preliminary work
+
+You'll need to complete these steps.
+
+=head3 Get list of active Core Team member emails
+
+Ask Rik Signes for an up-to-date list of active Core Team member emails from
+the TopicBox mailing list.
+
+B<TODO>: Replace this step with a machine-readable list of email addresses in
+C<Porting>.
```

Comment:
On the perl-core mailing list, there has been a discussion of the different email addresses which various committers use for different purposes.  Do we need to resolve that before we can address the B<TODO>?

Reply:
I believe #18941 resolves this.

Reply:
I will include a reference to `core-team.json` and optimistically assume that both PRs will eventually merge.